Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16-alpine3.14
```

Comment:
Why this Dockerfile is in this MR?

Reply:
Based on merge history, it seems it suggested the other branch instead of the main I desired. I'm reverting that commit right now.